python/python/glide/async_commands/core.py

```diff
@@ -2293,6 +2293,39 @@ async def zpopmax(
             ),
         )
 
+    async def bzpopmax(
+        self, keys: List[str], timeout: float
+    ) -> Optional[List[Union[str, float]]]:
+        """
+        Blocks the connection until it removes and returns a member with the highest score from the first non-empty
```

Comment:
should we write like blpop?
https://github.com/aws/glide-for-redis/blob/16a60ad974b4d4bd165e201a9828bc92b9c21f9a/python/python/glide/async_commands/core.py#L1129
https://github.com/aws/glide-for-redis/blob/16a60ad974b4d4bd165e201a9828bc92b9c21f9a/python/python/glide/async_commands/core.py#L1130

first what the command does and then the fact that its a blocking command?
if not, maybe we should change blpop and other blocking commands doc to have the same format

Reply:
Will update to align with blpop